cogs/player/jockey.py

```diff
@@ -632,3 +639,47 @@ async def update_now_playing(self):
                     self.guild.name,
                     exc
                 )
+
+    async def play_bump(self) -> bool:
```

Comment:
Minor, but I think a better pattern would be for this to return `None` and raise an exception instead if bump playback fails for any reason, such as the feature being disabled for a guild. I refactored Jockey for this exact reason in 89984b814ea2d1b138b1bf8403461f5c2323b171 😆

Feel free to create new exception class(es) in `utils/exceptions.py` for this purpose, as it looks like playback for bumps can fail for subtly different reasons which you may want to handle differently in `skip()`.